src/lib/settings/initialSettings.tsx

```diff
@@ -469,6 +471,19 @@ export function createSettings() {
           inputType: 'boolean',
         },
       }),
+      /**
+       * Use the new sketch mode implementation - solver (Dev only)
+       */
+      useNewSketchMode: new Setting<boolean>({
+        hideOnLevel: 'project',
+        hideOnPlatform: env().NODE_ENV === 'development' ? undefined : 'both', // Only show in dev mode
```

Comment:
Potentially you could also use the `IS_STAGING_OR_DEBUG` constant here if that's more semantically correct for what you're trying to accomplish.

Reply:
Cool!

Reply:
I left a comment about using Adam's endpoint instead too
[85425d3](https://github.com/KittyCAD/modeling-app/pull/8144/commits/85425d320e665580323aebd316660495bec3ba35)